<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QWidget" name="welcomePage">
     <widget class="QPushButton" name="buttonWelcomePlaygame">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>350</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>PLAYGAME</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonWelcomeLoadgame">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>350</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>LOADGAME</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonWelcomeExit">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>350</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelWelcomepageCentralname">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>591</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:48pt; font-weight:700;&quot;&gt;Human Resource Machine&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="fail">
     <widget class="QPushButton" name="buttonFailExit">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>320</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonFailRestart">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>320</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>RESTART</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonFailBack">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>320</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelFailCentral">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>160</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt; font-weight:700;&quot;&gt;FAIL&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="levelchoice">
     <widget class="QPushButton" name="buttonLevelchoiceLevel1">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>290</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Level 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonLevelchoiceLevel2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>290</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Level 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonLevelchoiceLevel3">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>290</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Level 3</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelLevelchoiceCentrallevelchoice">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>180</y>
        <width>291</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:48pt; font-weight:700;&quot;&gt;Level Choice&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonLevelchoiceBack">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>350</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="loadgame">
     <widget class="QPushButton" name="buttonLoadgameLoadlevelinfo">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>270</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>LOAD LEVELINFO</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonLoadgameLoadarchive">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>270</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>LOAD ARCHIVE</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelLoadgameCentral">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:700;&quot;&gt;Load Game&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonLoadgameBackandload">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>310</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonLoadgameAutoload">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>270</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>AUTO LOAD</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="playgame">
     <widget class="QTextEdit" name="inputPlaygameCommand">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>120</y>
        <width>141</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonPlaygameStart">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>70</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="showPlaygameLogbar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>141</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelPlaygameLogbar">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700;&quot;&gt;LOG&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="showPlaygameInboxbar">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>291</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="showPlaygameOutboxbar">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>390</y>
        <width>291</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="showPlaygameCarpet1">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>160</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="showPlaygameCarpet2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>160</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="showPlaygameCarpet3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>250</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="showPlaygameCarpet4">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>250</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelPlaygameInboxbar">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;InboxBar&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPlaygameOutboxbar">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>360</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;OutboxBar&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPlaygameCarpetbar">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>130</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Carpet&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="showPlaygameHand">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>190</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelPlaygameHand">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>170</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Hand&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="success">
     <widget class="QLabel" name="labelSuccessCentral">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>170</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt; font-weight:700; vertical-align:super;&quot;&gt;SUCCESS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSuccessBack">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>350</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSuccessExit">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>350</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHuman_Resource_Machine">
    <property name="title">
     <string>Human Resource Machine</string>
    </property>
   </widget>
   <addaction name="menuHuman_Resource_Machine"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
